### DISCUSSION TITLE
(Menu Bar Glitch?) Full-screen breaking custom menu bar

### DISCUSSION THREAD

**Discussion by Azielotl** [jordanbaird/Ice#198] [Q&A] [answered]
Hello.

I've noticed a glitch in the menu bar when activated upon full screen.

Upon activating the "split" option for the menu bar and going into full screen, it breaks the menu bar randomly and doesn't completely blacken out the menu bar on MacBooks. It also doesn't completely blacken out the menu bar on the "full" option.

https://github.com/jordanbaird/Ice/assets/70421893/c32916d7-e401-438f-aa42-325a957a5b97

Perhaps there could be a possibility to fix this issue (and maybe an option to disable custom appearance for full screen)?

**Comment by jordanbaird** [comment, marked as answer]
Try the [latest beta](https://github.com/jordanbaird/Ice/releases/tag/0.10.0-beta.6).  It should be fixed there.
